src/User32.Desktop/User32.cs

```diff
@@ -68,6 +69,16 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowLong(IntPtr hWnd, WindowLongIndexFlags nIndex);
 
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Auto)]
+        public static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName, int nMaxCount);
```

Comment:
We avoid `StringBuilder` in parameter types in favor of `char*` with a `[Friendly]` attribute. See [guidelines](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#method-parameter-types).